### ISSUE TITLE
Fan Controllers appear as dimmers

### ISSUE THREAD

**Issue by jimcortez** [marthoc/homeseer#27]
I have a couple of these in my house: https://shop.homeseer.com/products/z-wave-fan-control-switch

When bridging them from HS to HA, I see them show up as lights with a dimmer, instead of fans with specific levels.

Happy to provide any information that would be helpful.

**Comment by marthoc** [comment]
I’m going to guess that the device type string for these in HS is “Z-Wave Switch Multilevel” which would explain it. The default for these in the custom component is to be represented as a light. Can you verify that and also send me a screenshot of the control graphics page for that device? Should have three control functions: On/Off/Dim. I’ve already implemented support for importing certain ZWSMs as Covers, wouldn’t be hard to do the same for these as fans.

**Comment by jimcortez** [comment]
Is this what you are looking for? Homeseer calls it a "Z-Wave Switch Multilevel Root Device"

<img width="1164" alt="Screen Shot 2021-02-02 at 5 52 11 PM" src="https://user-images.githubusercontent.com/1000220/106686605-726e1100-657f-11eb-8463-d2fea85f22b2.png">

**Comment by marthoc** [comment]
The actual feature must be called "Z-Wave Switch Multilevel" or else the component wouldn't import it. HS has made it very difficult/obscured to find the "under-the-hood" API data in HS4.

Based on that page, it looks like the control is represented as a slider in HS4? It would appear to be the case that values from 1-33 set the fan to low, 34-66 to medium, and 67-99 to high?

**Comment by jimcortez** [comment]
You are correct, it is represented as a slider, which is a bit of an odd UX choice (like most things in the HS UI).

When I had this connected directly to the HA zwave implementation, it correctly picked up the High/Med/Low settings.

**Comment by marthoc** [comment]
I’m working on this - this is the first fan platform for the integration so I’m having to learn the fan platform!

**Comment by jimcortez** [comment]
Thanks! I am a big.... fan... of this integration! Already moved over 50 devices to it and the stability of my system has drastically improved.

**Comment by ScottGA** [comment]
Your integration is phenomenal!  I've been using Homeseer for a while now but never liked the available GUI options until I discovered your bridge to Home Assistant.  Now I have Lovelace running on all sorts of devices and it's fantastic.   The one gap I have is controlling fans.  Any update on implementing the fans as discussed above?  I have multiple Homeseer HSFC-200+ switches and they are bridged over to Home Assistant.  But they appear to be just On/Off switches in Home Assistant and not dimmable.  You were correct that HS reports them as "Z-Wave Switch Multilevel" Device Type (string).  I was trying to use the custom lovelace card (fan-percent-button-row), but the HA device needs to be dimmable .  I looked at some of the HS WD200+ Dimmer Switches bridged over from HS and they also have a "Z-Wave Switch Multilevel" Device Type (string).  So I'm not sure why the fan switches aren't being reflected as dimmable in Home Assistant.  Any suggestions?  Thanks!

**Comment by waynehead99** [comment]
@ScottGA I have noticed the same thing... and not sure how others are seeing the fan controllers as dimmable (not as pretty as an actual fan, but at least more functional than an on/off). 

I am still using HS3 vs HS4 and have no plans right now to update to HS4... curious if you are seeing the same and maybe this is a difference between the two versions?
